connectors/sources/github.py

```diff
@@ -2142,24 +2168,14 @@ async def get_docs(self, filtering=None):
                         else:
                             yield file_document, None
 
-                if path := rule["filter"].get(ObjectType.PATH.value):
-                    async for (
-                        file_document,
-                        attachment_metadata,
-                    ) in self._fetch_files_by_path(repo_name=repo_name, path=path):
-                        if file_document["type"] == FILE:
-                            attachment_metadata["url"] = attachment_metadata["git_url"]
-                            yield (
-                                file_document,
-                                partial(
-                                    self.get_content, attachment=attachment_metadata
-                                ),
-                            )
-                        else:
-                            yield file_document, None
```

Comment:
Why was this block removed?

Reply:
Omg it was a mistake, deleted accidentally. I'll return it back!